app/repositories/index.js

```diff
@@ -25,10 +26,12 @@ async function getById(mapDataset) {
 
 async function fetchById(mapDataset) {
   try {
-    const response = await fetch(external[mapDataset]);
+    const response = await fetch(`${external[mapDataset].url}?${external[mapDataset].query}`);
```

Comment:
Requires `query` to be defined on all the external datasets. We can run can JavaScript inside curly braces.
```javascript
const response = await fetch(`${external[mapDataset].url}?${external[mapDataset].query || ''}`);
```